components/TheHeader.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="header">
+    <div>
+      <img :src="navLogo" />
+      <span class="headerText">Todos</span>
+    </div>
+
+    <div class="flexBox">
+      <div id="searchIcon">
+        <img :src="seachIcon" />
+      </div>
+      <select class="" @change="switchLanguage">
```

Comment:
We can remove empty class declaration if not needed.

Reply:
Resolved